src/Uno.UI/UI/Xaml/Controls/ScrollBar/ScrollBar.cs

```diff
@@ -969,6 +972,20 @@ internal bool IsIgnoringUserInput
 		{
 			var value = Value;
 			var change = LargeChange;
+			if (VisualTreeHelper.GetParent(VisualTreeHelper.GetParent(VisualTreeHelper.GetParent(this))) is ScrollViewer
+			    sv)
```

Comment:
it seems fragile to navigate up exactly 3times, as this would be depending on the control template
you can just use this:
```suggestion
			if (this.FindFirstAncestor<ScrollViewer>() is { } sv) // using Uno.UI.Extensions;
```

Reply:
you can also use the TemplatedParent here
which should be the ScrollViewer instance
to avoid having to crawl the visual tree

Reply:
Yes, that was extremely hacky. I was just publishing the workaround to see if it works before cleaning out this mess :). I didn't know about TemplatedParent, so thanks for that. I still feel like fixing the problem in ScrollBar is the wrong way to go about this (it's ScrollViewer's responsibility to properly set its ScrollBars). I was thinking maybe we can set `ScrollBar.LargeChange` from ScrollViewer whenever `ActualHeight`/`ActualWidth` is changed or something. What do you think about doing it this way?

Reply:
not sure what it would be the responsibility of ScrollViewer here?
but looking at the code that was added, you can simply use this.Orientation to determine which scrollbar this is

Reply:
> not sure what it would be the responsibility of ScrollViewer here?
That's an outdated remark, ignore that. I initially thought the fix would be for the ScrollViewer to set SmallChange/LargeChange of its ScrollBar(s). 

Using this.Orientation is obviously more reasonable than that object.ReferenceEquals I added. I will replace it. My bad.